eslintrc.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * (c) 2010-present DEMOS plan GmbH.
+ *
+ * This file is part of the package demosplan,
+ * for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+// For better readability, all props are quoted in this file.
+/* eslint-disable quote-props */
+
+process.env.NODE_ENV = 'development'
+
+module.exports = {
+  root: true,
+  parserOptions: {
+    parser: '@babel/eslint-parser',
+    ecmaVersion: 6,
+    sourceType: "module",
+    ecmaFeatures: {
+      jsx: true
+    }
```

Comment:
i don't think we need the jsx support here since we are not using the syntax extension. From my perspective it is beneficial to keep the config as small as possible.

Reply:
true. May be a leftover when I tried to get TS running.